Trakt/Api/DataContracts/BaseModel/TraktEpisode.cs

```diff
@@ -1,30 +1,34 @@
 using System.Text.Json.Serialization;
 
-namespace Trakt.Api.DataContracts.BaseModel;
```

Comment:
Why move away from file-scoped space?
That's the newer way to do things

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces

I can re-clean it up in a future PR, Rider makes it very easy to do for the entire solution

Reply:
Sounds good. I just adapted the style from the main Jellyfin codebase...
I was already nearly done when I thought about looking up which style is current best practices :/